contracts/ICS20Transfer.sol

```diff
@@ -40,8 +40,10 @@ contract ICS20Transfer is IIBCApp, IICS20Transfer, IICS20Errors, Ownable, Reentr
     }
 
     /// @inheritdoc IICS20Transfer
-    function sendTransfer(SendTransferMsg calldata msg_) external override returns (uint32) {
-        return IICS26Router(owner()).sendPacket(ICS20Lib.newMsgSendPacketV1(_msgSender(), msg_));
+    function ibcERC20Contracts(string calldata denom) external view returns (address) {
```

Comment:
nit: should this be called ibcERC20Contract (minus the s, since it gets a single one)?